src/components/link/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import {
+  Link as ChakraLink,
+  type LinkProps as ChakraLinkProps,
+} from '@chakra-ui/react';
+
+export type LinkProps = ChakraLinkProps & {
```

Comment:
This originally included `iconLeft` and `iconRight` only to apply margin and pass them as children to link. We can leverage `flex` layout for spacing and don't need to overly complicate the recipe with parts for just that purpose